py-polars/tests/unit/test_datelike.py

```diff
@@ -56,9 +56,16 @@ def test_filter_date() -> None:
     assert df.filter(pl.col("date") <= pl.lit(datetime(2019, 1, 3))).is_empty()
     assert df.filter(pl.col("date") < pl.lit(datetime(2020, 1, 4))).shape[0] == 2
     assert df.filter(pl.col("date") < pl.lit(datetime(2020, 1, 5))).shape[0] == 3
-    assert df.filter(pl.col("date") <= pl.lit(datetime(2019, 1, 3))).is_empty()
-    assert df.filter(pl.col("date") < pl.lit(datetime(2020, 1, 4))).shape[0] == 2
-    assert df.filter(pl.col("date") < pl.lit(datetime(2020, 1, 5))).shape[0] == 3
+    assert df.filter(pl.col("date") <= pl.lit(date(2019, 1, 3))).is_empty()
+    assert df.filter(pl.col("date") < pl.lit(date(2020, 1, 4))).shape[0] == 2
+    assert df.filter(pl.col("date") < pl.lit(date(2020, 1, 5))).shape[0] == 3
```

Comment:
These `assert`s are identical to the three above. I assume the author intended to compare both `date` and `datetime`.